src/App/pages/LandingPage/Main/FilterFormDate/FilterFormDate.js

```diff
@@ -0,0 +1,11 @@
+export function FilterFormDate({ }) {
+    return (
+        <div className='filter-form'>
+                <select className='filter-form__selector' name="select" >
+                    {
+                            <option value='15.12.2021'>Date</option>
```

Comment:
Why is this a hardcoded date value? If you want this option to contain today's date, use JS Date object to get today's date and set it to the value property.

Reply:
fixed